tools/packaging/kata-deploy/scripts/kata-deploy.sh

```diff
@@ -721,6 +785,11 @@ function main() {
 			       fi
 			fi
 
+			if [ "$use_containerd_drop_in_conf_file" = "true" ]; then
+				mkdir -p $(dirname "/host/$containerd_drop_in_conf_file")
```

Comment:
As you have the leading `/` in the value or `containerd_drop_in_conf_file` I'm not sure you need another one. This does make it look much nicer though!